Assignment2/AlvaroMK-Alvaro Ajenjo Jurado-fromJSONtoJSON-LD.jsonld.txt

```diff
@@ -0,0 +1,34 @@
+{
+	"@context":{
+		"base": "http://example.org/",
+		"fullname":"base:fullname",
+		"hometown":"base:hometown",
+		"age":"base:age",
+		"almaMater":"base:almamater",
+		"parents":"base:parents",
+		"subjects":"base:subjects",
+		"nameSubjects":"base:nameSubjects",
+		"teacherSubjects":"base:teacherSubjects",
+		"DescriptionSubjects":"base:DescriptionSubjects",
+	},
+	"@id":"http://example.org/Alvaro",
+	"@type":"base",

```

Comment:
When defining a "@type" you have to specify what is the specific concept that represents the resource. In this case, using only "base" means that you are defining it as your prefix, but if you define it as "base:Person" you are specifying that the resource with "@id":"http://example.org/Alvaro" is a Person. 
Please, @AlvaroMK  be careful whit this kind of definition for next time.